internal/rule/ruleconfiguration/ruleconfiguration.go

```diff
@@ -3305,37 +3305,917 @@ var configurations = []Type{
 		ErrorModes:       []rulemode.Type{rulemode.Default},
 		RuleFunction:     rulefunction.PackageIndexPackagesHelpOnlineDeadLink,
 	},
+	{
+		ProjectType:      projecttype.PackageIndex,
+		SuperprojectType: projecttype.All,
+		Category:         "data",
+		Subcategory:      "package",
+		ID:               "IL001",
+		Brief:            "packages[].platforms missing",
+		Description:      "",
+		MessageTemplate:  "Missing packages[].platforms property in package(s): {{.}}",
+		DisableModes:     nil,
+		EnableModes:      []rulemode.Type{rulemode.Default},
+		InfoModes:        nil,
+		WarningModes:     nil,
+		ErrorModes:       []rulemode.Type{rulemode.Default},
+		RuleFunction:     rulefunction.PackageIndexPackagesPlatformsMissing,
+	},
+	{
+		ProjectType:      projecttype.PackageIndex,
+		SuperprojectType: projecttype.All,
+		Category:         "data",
+		Subcategory:      "platform",
+		ID:               "IL002",
+		Brief:            "incorrect packages[].platforms type",
+		Description:      "Must be a string.",
```

Comment:
Is this correct? Shouldn't `packages[].platforms` be a json array? :thinking:

Reply:
Thanks Silvano! All three fixed here:
https://github.com/arduino/arduino-lint/compare/6af76b178f491b9239c734d43bb09613c81f9e84..c1371a1e713cc0918c4ecb67da6d4fce0221fcdc